partner_communication_switzerland/controllers/employee_image.py

```diff
@@ -0,0 +1,54 @@
+##############################################################################
+#
+#    Copyright (C) 2024 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Noé Berdoz <nberdoz@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+import base64
+
+from werkzeug.exceptions import NotFound
+
+from odoo import http
+from odoo.http import request
+
+
+class EmployeeImageController(http.Controller):
+    @http.route(
+        "/employee/image/<int:employee_id>/",
+        auth="public",
+        type="http",
+        methods=["GET"],
+        website=True,
```

Comment:
I don't think we need the website parameter if it's a URL not used on the website